### PR TITLE
Create ServicePackage model

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #70**
- [x] Create `ServicePackage` model
- [x] Create `service_package` migration
- [x] Test migration

### CODE REVIEW

**Comment by sofiaromorales** [comment]
@Ovifer13 pls fix error

Screen Shot 2022-07-22 at 12 30 57 AM

File: pages/api/models/ServicePackage.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+module.exports = (sequelize: any) => {
+
+    class ServicePackage extends Model {}
```
Comment (sofiaromorales) [inline pages/api/models/ServicePackage.ts:3]: Import missing

**Review by sofiaromorales — CHANGES_REQUESTED** [review]

**Comment by Ovifer13** [comment]
> @Ovifer13 pls fix this error and resolve conflicts
> 
> Screen Shot 2022-07-22 at 12 30 57 AM

@sofiaromorales I don't have this error, also I tried to import `const { DataTypes, Model } = require('sequelize')` and It tells me that It's already defined in another file (For what I read is that TypeScript works like that)

File: pages/api/models/ServicePackage.ts (hunk 1 above)
Comment (Ovifer13) [reply pages/api/models/ServicePackage.ts:3]: If I do the import I get the following error
image

**Comment by Ovifer13** [comment]
**Implementation Proof**
image

**Review by sofiaromorales — APPROVED** [review]

**Pull request merged**